frontend/src/rest/ApiClient.js

```diff
@@ -0,0 +1,695 @@
+/* eslint-disable */
+/**
+ * NeuroScan
+ * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ *
+ * The version of the OpenAPI document: 1.0.0
+ * Contact: info@metacell.us
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ *
+ */
+
+
+import superagent from "superagent";
+import querystring from "querystring";
+
+/**
+* @module ApiClient
+* @version 1.0.0
+*/
+
+/**
+* Manages low level client-server communications, parameter marshalling, etc. There should not be any need for an
+* application to use this class directly - the *Api and model classes provide the public API for the service. The
+* contents of this file should be regarded as internal but are documented for completeness.
+* @alias module:ApiClient
+* @class
+*/
+class ApiClient {
+    constructor() {
+        /**
+         * The base URL against which to resolve every API call's (relative) path.
+         * @type {String}
+         * @default http://localhost:1337
+         */
+        this.basePath = 'http://localhost:1337'.replace(/\/+$/, '');
+
+        /**
+         * The authentication methods to be included for all API calls.
+         * @type {Array.<String>}
+         */
+        this.authentications = {
+            'bearerAuth': {type: 'bearer'} // JWT
+        }
+
+        /**
+         * The default HTTP headers to be included for all API calls.
+         * @type {Array.<String>}
+         * @default {}
+         */
+        this.defaultHeaders = {};
```

Comment:
Also changed the generated code here due to 'Ajax request: Refused to set unsafe header'
Original code was:
        this.defaultHeaders = {
            'User-Agent': 'OpenAPI-Generator/1.0.0/Javascript'
        };

Reply:
Here we should use `process.env.REACT_APP_BACKEND_URL` instead of hardcoded localhost.

Reply:
I agree. However, I don't seem to have access to the process.env variable in the frontend. 
![image](https://user-images.githubusercontent.com/19196034/126767298-fb2aa9b4-c104-4006-8dd0-6d31d190162e.png)
I will look into how to have environment variables in the frontend

Reply:
@afonsobspinto  it's resolved at build time, you cannot access process during runtime

You can add a `.env.local` to the frontend project like this:
<img width="419" alt="Bildschirmfoto 2021-07-23 um 12 11 32" src="https://user-images.githubusercontent.com/7281685/126768084-133261ee-b6e7-493f-9bc1-5ffdabb702fd.png">

I will add this to the README as well, should have been stated there

Reply:
Works. Thanks